WalletWasabi.Fluent/ViewModels/Wallets/Send/PrivacySuggestionsFlyoutViewModel.cs

```diff
@@ -34,9 +34,10 @@ public PrivacySuggestionsFlyoutViewModel()
 
 	public ObservableCollection<SuggestionViewModel> Suggestions { get; }
 
-	public async Task BuildPrivacySuggestionsAsync(Wallet wallet, TransactionInfo info, BitcoinAddress destination, BuildTransactionResult transaction, bool isFixedAmount)
+	public async Task BuildPrivacySuggestionsAsync(Wallet wallet, TransactionInfo info, BitcoinAddress destination, BuildTransactionResult transaction, bool isFixedAmount, CancellationToken cancellationToken)
 	{
-		using CancellationTokenSource cancellationTokenSource = new(TimeSpan.FromSeconds(15));
+		using CancellationTokenSource childCTS = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken);
+		childCTS.CancelAfter(15000);
```

Comment:
Wow, I've never used this, awesome!

Reply:
But [03c0f20](https://github.com/zkSNACKs/WalletWasabi/pull/7192/commits/03c0f202090b6fc74b496052730922c9c67785b3) changed this already, how isn't github showing it...